build.sbt

```diff
@@ -58,6 +57,7 @@ lazy val root = (projectMatrix in file("."))
     name := "atum-service-root",
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint"),
     publish / skip := true,
+    crossScalaVersions := Nil,
```

Comment:
What is the purpose of this setting?

Reply:
I think I took it from the sbt documentation: https://www.scala-sbt.org/1.x/docs/Cross-Build.html

`crossScalaVersions must be set to Nil on the aggregating project`

so this is Nil here only for the root one

Reply:
But you are using projectmatrix, not the stateful build. So using `crossScalaVersions` makes no sense or am I missing something?